routing/routes.go

```diff
@@ -27,6 +27,18 @@ func NewRoutes() (r *Routes) {
 			"/api",
 			handlers.Root,
 		},
+		Route{
+			"ServiceList",
+			"GET",
+			"/api/services",
+			handlers.ServiceList,
+		},
+		Route{
+			"ServiceShow",
+			"GET",
+			"/api/namespaces/{namespace_id:[a-z]+}/services/{id:[a-z]+}",
```

Comment:
We need to change {namespace_id:[a-z]+} because the patter does not cover all valid namespaces.  For example, "istio-system" won't match here.

I think we could just omit the pattern here unless you want to define something very broad, because I think it can be any valid DNS label.  So, maybe just:

`"/api/namespaces/{namespace_id}/services/{id:[a-z]+}"`

I'm not actually sure about the service names, but we may need to ensure the pattern is valid for service names as well.

Reply:
@jshaughn you are right :) I already changed this! I also changed the pattern for service ids since they follow same syntax as namespaces.